File: src/pixelHat/RedBlueMovie.java
package pixelHat;

import java.io.PrintStream;

public class RedBlueMovie {

	public static void main(String[] args) throws Exception {
			
		SpriteLoader sprites = new SpriteLoader();
		
		PrintStream ps = new PrintStream("redBlueGEN.txt");
		
		int [] letterMap =      {'#', 1};
		
		int [][] letterG = sprites.colorSprite("letterG", letterMap);
		int [][] lettero = sprites.colorSprite("lettero", letterMap);
				
		int [][] five = sprites.colorSprite("five", letterMap);
		int [][] eight = sprites.colorSprite("eight", letterMap);
		
		for(int i=0;i<10;++i) {
		for(int x=0;x<63;x=x+2) {
			HatFrame f = new HatFrame();
			f.drawSprite(x+0, 2+1, letterG);
			f.drawSprite(x+13, 2+5, lettero);
			f.drawSprite(x+23, 2+0, five);
			f.drawSprite(x+33, 2+0, eight);
			f.drawSprite(x+43, 2+0, five);
			f.drawSprite(x+53, 2+0, eight);
						
			ps.println("%");
			ps.println(f.toString());
		}
		}	
		
		ps.println();
		ps.flush();
		ps.close();			
		
	}

}

File: pysrc/RedBlueMovie.py
import SpriteLoader
import HatFrame

sprites = SpriteLoader.SpriteLoader()

letterMap = ['#', 1]

letterG = sprites.colorSprite("letterG", letterMap)
lettero = sprites.colorSprite("lettero", letterMap)
                
five = sprites.colorSprite("five", letterMap)
eight = sprites.colorSprite("eight", letterMap)

with open("redBlueGEN.txt","w") as ps:
    for i in range(0,10):
        for x in range(0,63,2):
            f = HatFrame.HatFrame()
            f.draw_sprite(x+0,2+1,letterG)
            f.draw_sprite(x+13, 2+5, lettero)
            f.draw_sprite(x+23, 2+0, five)
            f.draw_sprite(x+33, 2+0, eight)
            f.draw_sprite(x+43, 2+0, five)
            f.draw_sprite(x+53, 2+0, eight)
            
            ps.write("%\n")
            ps.write(f.to_string()+"\n")
    
    ps.write("\n")        